modular_skyrat/code/modules/jobs/job_types/brig_physician.dm

```diff
@@ -10,7 +10,7 @@
 	selection_color = "#c02f2f"
 	minimal_player_age = 7
 	exp_requirements = 120 //SKYRAT CHANGE - lowers security exp requirement
```

Comment:
Fix this line to say "medical exp requirement"

Reply:
addressed in third commit